include/swift/SIL/SILBridging.h

```diff
@@ -446,6 +473,23 @@ struct BridgedGenericSpecializationInformation {
   const swift::GenericSpecializationInformation * _Nullable data = nullptr;
 };
 
+struct OptionalBridgedSILDebugVariable {
+  uint64_t storage[16];
+
+#ifdef USED_IN_CPP_SOURCE
+  using OptionalSILDebugVariable = llvm::Optional<swift::SILDebugVariable>;
+
+  OptionalBridgedSILDebugVariable(
+    OptionalSILDebugVariable &&debugVariable) {
+    static_assert(sizeof(OptionalSILDebugVariable) <= 16*8);
```

Comment:
why not `static_assert(sizeof(OptionalSILDebugVariable) <= sizeof(OptionalBridgedSILDebugVariable))`?